tests/unit/lms/services/blackboard_api/_basic_test.py

```diff
@@ -51,8 +51,8 @@ def test_when_the_response_object_is_None(self):
         assert BlackboardErrorResponseSchema(None).parse() == {}
 
 
-class TestBasicClient:
-    def test_get_token(self, basic_client, oauth_http_service):
+class TestGetToken:
```

Comment:
```suggestion
class TestBasicClientGetToken:
```

As there are tests for different classes in here (`TestBlackboardErrorResponseSchema`) this is kind of confusing, as we are swapping between two different styles.

Reply:
:+1: Done